### ISSUE TITLE
Swipe box doesn't work with jQuery 3.2.1

### ISSUE THREAD

**Issue by icekem** [brutaldesign/swipebox#352]
Swipe box doesn't work with jQuery 3.2.1.

When you click any space on page black screen opens. Can you fix it?

**Comment by shawnCaza** [comment]
Can 

`selector = elem.selector,`

just be replaced with:

`selector = '.swipeboxImg',`

**Comment by shandhiviyarajan** [comment]
no i tried this but i still showing the black screen,

**Comment by shawnCaza** [comment]
Yes, that would be an issue. Do you know where custom class names are set?

**Comment by woodsba** [comment]
Normally when the .swipebox is called.

For example in mine it is $('.swipebox').swipebox({options});

**Comment by zuera** [comment]
The solution of @shandhiviyarajan worked for me.

**Comment by woodsba** [comment]
Yes that works in that one instance, but What if you have different selectors to trigger it with different options, like you want one swipebox on your page looping, and another not.

**Comment by risingisland** [comment]
Does anybody have a working update that they would be willing to post?

**Comment by bmcclure** [comment]
Here's a patch file we've been using internally to make Swipebox work for us. I think this should work regardless of the selector, but let me know if you see any issues with it.

[swipebox-selector-fix.patch.txt](https://github.com/brutaldesign/swipebox/files/2081453/swipebox-selector-fix.patch.txt)

**Comment by bmcclure** [comment]
Actually, this PR is an even better fix I think: https://github.com/brutaldesign/swipebox/pull/298

**Comment by peterjosvai** [comment]
> Here's a patch file we've been using internally to make Swipebox work for us. I think this should work regardless of the selector, but let me know if you see any issues with it.
> 
> [swipebox-selector-fix.patch.txt](https://github.com/brutaldesign/swipebox/files/2081453/swipebox-selector-fix.patch.txt)

bmcclure! thanks a LOT...
I tried the first suggestion, by shawnCaza, and it didn't work, I could click on the picture already, but that would only cause the referenced image to be downloaded (shown) directly, by its URL

the patch, however, works perfectly...
I've never seen a patch file like this before, totally love it! :)

thanks a lot @ 02.22.2020
